webui/src/pages/setup/index.jsx

```diff
@@ -75,15 +61,6 @@ const SetupContents = () => {
     switch (currentStep) {
         case SETUP_STATE_INITIALIZED:
             return router.push({pathname: '/', query: router.query});
-        case SETUP_STATE_COMMUNICATION_PERFS_DONE:
```

Comment:
If communication preferences are disabled server-side, this will be the initial state, so you can't remove this case unless you also handle this change server-side.

Reply:
ack.